test/unit/org/apache/cassandra/db/lifecycle/LogTransactionTest.java

```diff
@@ -25,6 +25,7 @@
 import java.util.*;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
+import java.util.function.Function;
```

Comment:
nit: unused

In general, the imports are a little wacky here w/ the static in the middle :/